src/main/java/org/ibp/api/ibpworkbench/rest/BreedingView.java

```diff
@@ -17,23 +17,28 @@
 import org.ibp.api.ibpworkbench.constants.WebAPIConstants;
 import org.ibp.api.ibpworkbench.model.DataResponse;
 import org.ibp.api.ibpworkbench.service.BreedingViewService;
+import org.ibp.api.java.design.DesignLicenseService;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
 import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestMethod;
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.ResponseBody;
 import org.springframework.web.bind.annotation.RestController;
 
+import javax.annotation.Resource;
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
-@Api(value = "Breeding View", description = "Web Services to process the Breeding View output")
+@Api(value = "Breeding View Resource")
 @RestController
 @RequestMapping("/breeding_view")
```

Comment:
can we define breeding_view as a resource itself? it the answer is YES, then it should be called breeding-views instead.